molecule/_resources/destroy-lxd.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+- name: Destroy
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  no_log: "{{ not lookup('env', 'MOLECULE_DEBUG') | bool }}"
+  tasks:
+    - name: Destroy molecule instance(s)
+      lxd_container:
+        name: "{{ item.name }}"
+        state: absent
+        force_stop: "{{ item.force_kill | default(true) }}"
```

Comment:
`item.force_kill/item.force_stop` is the only difference with bundled LXD destroy playbook. Could not this bundled playbook be reused here ?

Reply:
True.
Unfortunately when I tried to remove the destroy plabooks Molecule did not work, and this is due to:
`If the determined driver has a key in the playbooks dict, Molecule will use this dict to resolve all provisioning playbooks (create/destroy).`
This is quite unfortunate.